### DISCUSSION TITLE
Youtube

### DISCUSSION THREAD

**Discussion by S0vereignity** [Prodigy-Hacking/ProdigyMathGameHacking#1229] [Questions] [answered]
Hey, when I looked at the channel for PHEx, it was **removed** from youtube.
Will you guys ever make a new channel?

**Comment by donaldli2020** [comment]
looks like it's deleted for good

**Reply by S0vereignity** [reply]
yea...

**Reply by afkvido** [reply]
Actually, no. They made a new channel https://www.youtube.com/watch?v=OhfHNatEtw4

**Reply by donaldli2020** [reply]
that was after my answer

**Comment by afkvido** [comment, marked as answer]
@GodlyGavin1 https://www.youtube.com/watch?v=OhfHNatEtw4

**Reply by S0vereignity** [reply]
Thanks
